atk-sys/Cargo.toml

```diff
@@ -18,6 +18,7 @@ tempfile = "3"
 v2_30 = []
 v2_32 = ["v2_30"]
 v2_34 = ["v2_32"]
+v2_35 = ["v2_34"]
```

Comment:
Huh. This is an unstable development version and should not really be here at all :) Did Ubuntu get ahead of themselves again and shipped that with a stable release?

Reply:
So we remove `atk_plug_set_child` ?

Reply:
Yes and the feature flag